firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionEvent.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.sessions
+
+/**
+ * Contains the relevant information around an App Quality Session.
+ *
+ * See go/app-quality-unified-session-definition for more details. Keep in sync with
+ * https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseSessions/ProtoSupport/Protos/sessions.proto
+ */
+// TODO(mrober): Add and populate all fields from sessions.proto
+internal data class SessionEvent(
+  /** The type of event being reported. */
+  val eventType: EventType,
```

Comment:
Do we need to remember this state? It is already a part of SessionStartEvent, can we use it from there? I'm trying to avoid multiple source of truth here.

Reply:
This is there to match the [proto](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseSessions/ProtoSupport/Protos/sessions.proto) 1:1, this data class will be serialized and sent to firelog.